AllReadyApp/Web-App/AllReady/wwwroot/js/Admin/Campaign/Edit/EditModule.js

```diff
@@ -0,0 +1,50 @@
+﻿/// <reference path="../../../../lib/jquery/dist/jquery.js" />
+"use strict";
+
+var editModule = (function () {
+
+    function deleteCampaignImageButtonClick() {
+
+        function deleteCampaignImageCallBack(e) {
+            e.preventDefault();
+
+            var confirmAnswer = confirm("Are you sure you want to delete the image?");
+
+            if (!confirmAnswer) return;
+
+            var campaignId = $("#Id").val();
+            var antiForgeryToken = $('input[name="__RequestVerificationToken"]').val();
+
+            $.ajax({
+                type: "POST",
+                url: "/Admin/Campaign/DeleteCampaignImage",
+                data: { __RequestVerificationToken: antiForgeryToken, campaignId: campaignId },
+                dataType: "json",
+                success: function (response) {
+
+                    if (response.status === "Success") {
+
+                        $("#image-panel-container").slideUp(1000);
+
+                        $(".image-delete-success-alert").slideDown(2000, function() {
+                            $(this).slideUp(3000);
+                        });
+                    }
+
+                },
+                error: function (xhr, status, error) {
+                    //console.log(xhr.responseText);
```

Comment:
If the commented code isn't needed, it can be removed. Otherwise, add back the logging or use a display on the page to let the user know the operation failed (the Bootstrap Alert is a good option for that).